test/techs.e2e-spec.ts

```diff
@@ -104,6 +104,38 @@ describe("Techs Controller (e2e)", () => {
                     );
                 });
         });
+        it("should return 404 if invalid teamId provided", async () => {
```

Comment:
re: [your comment](https://github.com/chingu-x/chingu-dashboard-be/pull/174#issuecomment-2213376203)

for example here, you added 401 and 403 but I noticed it's not already on swagger. This particular endpoint only has 200 under responses

![image](https://github.com/chingu-x/chingu-dashboard-be/assets/6191116/40f1e405-8e1d-4a74-aa38-cba18ad9a39b)

Reply:
Okay I will add these responses.